src/stores/__tests__/bots.spec.js

```diff
@@ -0,0 +1,92 @@
+import { setActivePinia, createPinia } from 'pinia'
+
+import {
+	getConversationBots,
+	disableBotForConversation,
+	enableBotForConversation,
+} from '../../services/botsService.js'
+import { generateOCSResponse } from '../../test-helpers.js'
+import { useBotsStore } from '../bots.js'
+
+jest.mock('../../services/botsService', () => ({
+	getConversationBots: jest.fn(),
+	disableBotForConversation: jest.fn(),
+	enableBotForConversation: jest.fn(),
+}))
+
+describe('botsStore', () => {
+	const token = 'TOKEN'
+	const bots = [
+		{ id: 1, name: 'Dummy bot 1', state: 0 },
+		{ id: 2, name: 'Dummy bot 2', state: 1 },
+	]
+	let botsStore
+
+	beforeEach(async () => {
+		setActivePinia(createPinia())
+		botsStore = useBotsStore()
+	})
+
+	afterEach(async () => {
+		jest.clearAllMocks()
+	})
+
+	it('returns an empty array when bots are not loaded yet for conversation', async () => {
+		// Assert: check initial state of the store
+		expect(botsStore.getConversationBots(token)).toEqual([])
+	})
+
+	it('processes a response from server and stores bots', async () => {
+		// Arrange
+		const response = generateOCSResponse({ payload: bots })
+		getConversationBots.mockResolvedValueOnce(response)
+
+		// Act: load bots from server
+		await botsStore.loadConversationBots(token)
+
+		// Assert
+		expect(getConversationBots).toHaveBeenCalledWith(token)
+		expect(botsStore.getConversationBots(token)).toEqual(bots)
+	})
```

Comment:
I didn't mean that we must literally add `// A*` comments =D

(But fine by me)

Reply:
I feel it to be more readable / understandable for the following development